packages/core/test/metadataStream.test.ts

```
@@ -0,0 +1,978 @@
+    // Should throw due to abort
+    await expect(metadataStream.wait()).rejects.toThrow();
+
+    // Request should have been made before abort
+    expect(requestReceived).toBe(true);
+  });
+
+  it("should handle empty stream (no chunks)", async () => {
+    async function* generateChunks() {
+      // Yields nothing
+      return;
+    }
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)